docs/source/app-dev/services.rst

```diff
@@ -197,6 +197,23 @@ See :ref:`transaction-filter` above.
 
   You can read more about offsets in the `protobuf documentation of the API <../app-dev/grpc/proto-docs.html#ledgeroffset>`__.
 
+.. event-query-service:
+
+Event Query Service (EXPERIMENTAL)
+==================================
+
+Use the **event query service** to obtain a party-specific view of contract related events.
+
+Contract events can be queried by contract-id or contract-key. If the events being queried are not visible to the requesting parties then an empty structure is returned. This service is will return consumed contracts up until the point they are pruned.
```

Comment:
Are contract-id and contract-key specific terminology? Normally neither term would have a hyphen (so, "contract id" and "contract key").

Reply:
Hyphens removed.